FrontEnd/src/components/ProfilePage/FormComponents/AdditionalInfo.js

```diff
@@ -14,6 +16,28 @@ const AdditionalInfo = (props) => {
     const { profile: mainProfile, mutate: profileMutate } = useProfile();
     const [profile, setProfile] = useState(props.profile);
     const [foundationYearError, setFoundationYearError] = useState(null);
+    const { setFormIsDirty } = useContext(DirtyFormContext);
+
+    // TODO: update default values as new fields added
```

Comment:
Do we need this todo?

Reply:
We haven't final profile model with all the fields still. New fields will be added soon or later. I think it's okay to leave this comment as a reminder that default values should be updated then.